configs/.env

```diff
@@ -0,0 +1,19 @@
+# GITHUB_ACCESS_TOKEN=
+
+# Runnable Test
+GITHUB_USERNAME=RunnableTest
+GITHUB_OAUTH_ID=17654916
+
+# Runnable
+# GITHUB_USERNAME=Runnable
+# GITHUB_OAUTH_ID=2828361
+
+# Gamma
+USER_CONTENT_DOMAIN=runnable.ninja
+API_URL=https://api.runnable-gamma.com
```

Comment:
Left `.env` in there with empty ENVs. Moved all other to `.env.test` which will be ignored.